addon/components/lazy-render-wrapper.js

```diff
@@ -206,6 +206,7 @@ export default Ember.Component.extend({
 				} else {
 					this.set('_hasUserInteracted', true);
 					this.set('_shouldShowOnRender', true);
+					this.set('_isInProcessOfShowing', true);
```

Comment:
So I've been thinking about our using of private variables on components given the other less-used alternative that is truly private variables in the file. For example:

```js
import Ember from 'ember';

let isInProcessOfShowing = false;

export default Ember.Component.extend({

  ...

   someMethod() {
      ...

      isInProcessOfShowing = true;
   }

});
```

What is your preferred approach? The example above would probably not work is we need to use `_isInProcessOfShowing` in other modules (e.g. a component that extends `lazy-render-wrapper`) so it may not be feasible. However, I would like to at least answer the question of which method we prefer.